tests/integration/test_types_integration.py

```diff
@@ -0,0 +1,208 @@
+import math
+import pytest
+from decimal import Decimal
+import datetime
+from datetime import timezone, timedelta
+import trino
+
+
+@pytest.fixture
+def trino_connection(run_trino):
+    _, host, port = run_trino
+
+    yield trino.dbapi.Connection(
+        host=host, port=port, user="test", source="test", max_attempts=1
+    )
+
+
+def test_int_types(trino_connection):
```

Comment:
I would split into test methods per type. `test_boolean`, `test_boolean` and so on.

Reply:
Done